packages/booter-lb3app/src/lb3app.booter.ts

```diff
@@ -92,9 +92,12 @@ export class Lb3AppBooter implements Booter {
     const swaggerSpec = generateSwaggerSpec(lb3App, {
       generateOperationScopedModels: true,
     });
-    const result = await swagger2openapi.convertObj(swaggerSpec, {
+    const fixedSwaggerSpec = JSON.parse(JSON.stringify(swaggerSpec));
```

Comment:
Can we add some comments here to explain why we need to force plain JSON?

Reply:
added